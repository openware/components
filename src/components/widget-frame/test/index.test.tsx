import * as React from 'react';
import { shallow } from 'enzyme';

import { WidgetFrame } from '..';

describe('WidgetFrame', () => {
    it('snapshot render', () => {
        expect(
            shallow(
                <WidgetFrame header="title" className="className">
                    Content
                </WidgetFrame>
            )
        ).toMatchSnapshot();
    });
});
