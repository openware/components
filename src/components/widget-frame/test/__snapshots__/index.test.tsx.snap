// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WidgetFrame snapshot render 1`] = `
<div
  className="widget-frame className"
>
  <div
    className="widget-frame__title"
  >
    <div
      className="widget-frame__title-text"
    >
      title
    </div>
  </div>
  <div
    className="widget-frame__children"
  >
    Content
  </div>
</div>
`;
